### PR TITLE
Add support for workflow usage and workflow run usage API endpoints

### PR DESCRIPTION
**Pull request by joshuabezaleel**
Fixes: #1526 

Add methods and tests for :
- [x] Workflow usage
- [x] Workflow run usage

### CODE REVIEW

**Comment by joshuabezaleel** [comment]
I still have some hesitancy on how should I name the structs since I am not that familiar with GitHub actions workflow and workflow run so I am sorry if I got some wrong. Looking forward to hearing your review @gmlewis ! :slightly_smiling_face:

File: github/actions_workflow_runs.go [hunk 1]
```diff
@@ -54,6 +54,25 @@ type ListWorkflowRunsOptions struct {
 	ListOptions
 }
 
+// WorkflowRunUsage represents a usage of a specific workflow run.
+type WorkflowRunUsage struct {
+	Billable    *WorkflowRunEnvironment `json:"billable,omitempty"`
+	RunDuration *int64                  `json:"run_duration_ms,omitempty"`
```
Comment (gmlewis) [inline github/actions_workflow_runs.go:60]: `RunDuration` => `RunDurationMS`

File: github/actions_workflows.go [hunk 2]
```diff
@@ -86,3 +103,36 @@ func (s *ActionsService) getWorkflow(ctx context.Context, url string) (*Workflow
 
 	return workflow, resp, nil
 }
+
+// GetWorkflowByID gets a specific workflow usage by ID in the unit of billable minutes.
```
Comment (gmlewis) [inline github/actions_workflows.go:107]: `GetWorkflowByID` => `GetWorkflowUsageByID`

`minutes` => `milliseconds`

File: github/actions_workflows.go [hunk 3]
```diff
@@ -86,3 +103,36 @@ func (s *ActionsService) getWorkflow(ctx context.Context, url string) (*Workflow
...
+//
+// GitHub API docs: https://developer.github.com/v3/actions/workflows/#get-workflow-usage
+func (s *ActionsService) GetWorkflowUsageByID(ctx context.Context, owner, repo string, workflowID int64) (*WorkflowUsage, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/actions/workflows/%v/timing", owner, repo, workflowID)
+
+	return s.getWorkflowUsage(ctx, u)
+}
+
+// GetWorkflowByFileName gets a specific workflow usage by file name in the unit of billable minutes.
```
Comment (gmlewis) [inline github/actions_workflows.go:116]: `GetWorkflowByFileName` => `GetWorkflowUsageByFileName`

`minutes` => `milliseconds`

File: github/actions_workflow_runs.go [hunk 4]
```diff
@@ -193,3 +212,23 @@ func (s *ActionsService) DeleteWorkflowRunLogs(ctx context.Context, owner, repo
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// GetWorkflowRunUsage gets a specific workflow usage run by run ID in the unit of billable minutes.
```
Comment (gmlewis) [inline github/actions_workflow_runs.go:216]: `GetWorkflowRunUsage` => `GetWorkflowRunUsageByID`

`minutes` => `milliseconds`

**Comment by joshuabezaleel** [comment]
So sorry that I missed the typos. Thank you very much for the review @gmlewis @wesleimp ! :slightly_smiling_face:

File: github/actions_workflow_runs.go [hunk 5]
```diff
@@ -54,6 +54,25 @@ type ListWorkflowRunsOptions struct {
 	ListOptions
 }
 
+// WorkflowRunUsage represents a usage of a specific workflow run.
+type WorkflowRunUsage struct {
+	Billable    *WorkflowRunEnvironment `json:"billable,omitempty"`
+	RunDuration *int64                  `json:"run_duration_ms,omitempty"`
+}
+
+// WorkflowRunEnvironment represents different runner environments available for workflow run.
```
Comment (nightlark) [inline github/actions_workflow_runs.go:63]:
```suggestion
// WorkflowRunEnvironment represents different runner environments available for a workflow run.
```

File: github/actions_workflow_runs.go [hunk 6]
```diff
@@ -54,6 +54,25 @@ type ListWorkflowRunsOptions struct {
...
+type WorkflowRunEnvironment struct {
+	Ubuntu  *WorkflowRunBill `json:"UBUNTU,omitempty"`
+	MacOS   *WorkflowRunBill `json:"MACOS,omitempty"`
+	Windows *WorkflowRunBill `json:"WINDOWS,omitempty"`
+}
+
+// WorkflowRunBill specifies a bill for a specific environment of workflow run.
```
Comment (nightlark) [inline github/actions_workflow_runs.go:70]:
```suggestion
// WorkflowRunBill specifies billable time for a specific environment in a workflow run.
```

**Pull request merged**